<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseProvidersPage</class>
 <widget class="QWidget" name="ChooseProvidersPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>440</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="providersBox">
     <property name="title">
      <string>Select Collections to Synchronize</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListView" name="providersView">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="matchLabel">
        <property name="text">
         <string notr="true">Tracks matched by: &lt;placeholder&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fieldsBox">
     <property name="title">
      <string>Select Fields to Synchronize</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="format">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>qdialogbuttonbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
